File: bootstrap/src/test/java/net/dontdrinkandroot/wicket/bootstrap/test/SignOutPageTest.java
package net.dontdrinkandroot.wicket.bootstrap.test;

import net.dontdrinkandroot.wicket.bootstrap.page.SignOutPage;
import org.apache.wicket.request.mapper.parameter.PageParameters;
import org.junit.Test;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
public class SignOutPageTest extends AbstractWicketTest
{
    @Test
    public void testInstantiation()
    {
        this.tester.startPage(new SignOutPage(new PageParameters()));
        this.tester.assertRenderedPage(SignOutPage.class);
    }
}

File: bootstrap/src/test/kotlin/page/SignOutPageTest.kt
package page

import net.dontdrinkandroot.wicket.bootstrap.page.SignOutPage
import net.dontdrinkandroot.wicket.bootstrap.test.AbstractWicketTest
import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test

class SignOutPageTest : AbstractWicketTest() {

    @Test
    @Disabled("Session in validate in Page causes this test to fail")
    fun testInstantiation() {
        tester.startPage(SignOutPage())
        tester.assertRenderedPage(SignOutPage::class.java)
    }
}